python/dolfinx/io/utils.py

```diff
@@ -276,8 +276,15 @@ def read_mesh(
         )
         return Mesh(msh, domain)
 
-    def read_meshtags(self, mesh, name, xpath="/Xdmf/Domain"):
-        mt = super().read_meshtags(mesh._cpp_object, name, xpath)
+    def read_meshtags(self, mesh, name, attribute_label=None, xpath="/Xdmf/Domain"):
```

Comment:
Follow the Google docs style with `Arguments`, etc. There are quite a few examples in the code.